services/ui-src/src/verbiage/pages/home.ts

```diff
@@ -1,6 +1,59 @@
 export default {
   intro: {
-    header: "MFP",
-    body: {},
+    header: "Money Follows the Person (MFP) Portal",
+    body: {
+      preLinkText:
+        "Get started by completing a MFP Work Plan for your state or territory. After your Work Plan is submitted and approved, you will be able to complete a Semi-Annual Progress Report. Learn more about this",
+      linkText: "new data collection tool",
+      linkLocation:
+        "https://www.medicaid.gov/medicaid/managed-care/guidance/medicaid-and-chip-managed-care-reporting/index.html",
+      postLinkText: " from CMS.",
+    },
+  },
+  cards: {
+    MP: {
+      title: "MFP Work Plan (WP)",
+      body: {
+        available:
+          "The MFP Work Plan online form is now available. Note: Every state must submit one report per program, through the online form.",
+      },
+      downloadText: "",
+      link: {
+        text: "Enter Work Plan online",
+        route: "mp/get-started",
```

Comment:
Question: should this route be `mfp/get-started` or `wp/get-started`?

Reply:
I think this brings up another question:
Based on the mcr reference below, are there two different get-started routes? One for Work Plan and one for SARS?

```cards: {
    MP: {
      title: "MFP Work Plan (WP)",
      body: {
        available:
          "The MFP Work Plan online form is now available. Note: Every state must submit one report per program, through the online form.",
      },
      downloadText: "",
      link: {
        text: "Enter Work Plan online",
        route: "mp/get-started",
      },
      accordion: {
        buttonLabel: "When is the Work Plan due?",
        text: "The initial Work Plan will be submitted with the updated operational protocol for review and approval. The Work Plan will be updated annually during Period 1 of SAR reporting (July-August). Recipients may also submit update to the WP during period 2 of SAR reporting (January - February) to the new initiatives or reflect any planned changes in state or territory-specific initiatives.",
      },
    },```